subworkflows/nf-core/fastq_align_chromap/meta.yml

```diff
@@ -0,0 +1,98 @@
+name: "fastq_align_chromap"
+description: Align high throughput chromatin profiles using Chromap then sort with samtools
+keywords:
+  - align
+  - fasta
+  - genome
+  - reference
+  - chromatin profiles
+  - chip-seq
+  - atac-seq
+  - hic
+modules:
+  - chromap/chromap
+  - samtools/sort
+  - samtools/index
+  - samtools/stats
+  - samtools/idxstats
+  - samtools/flagstat
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - ch_reads:
+      type: file
+      description: |
+        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+        Structure: [ val(meta), path(reads) ]
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing reference information
+        e.g. [ id:'test' ]
+  - ch_index:
+      type: file
+      description: |
+        Chromap genome index files
+        Structure: [ val(meta2), path(index) ]
+      pattern: "*.index"
+  - ch_fasta:
+      type: file
+      description: |
+        Reference fasta file
+        Structure: [ val(meta2), path(fasta) ]
+      pattern: "*.{fasta,fa}"
+  - ch_barcodes:
+      type: file
+      description: Cell barcode files
+      Structure: [path(barcodes)]
```

Comment:
Shouldn't be a separate entry! (I think missing a `|` and some spaces), check everywhere :+1:

Reply:
Fixed here [62e2b76](https://github.com/nf-core/modules/pull/2414/commits/62e2b76fce46bb63eb5de4a836b20c01e08af534)